modules/moderation.py

```diff
@@ -437,7 +437,14 @@ async def _strike_set(self, ctx, member: discord.Member, count: StrikeRange, *,
                         'strike_check': time.time() + (60 * 60 * 24 * 7)
                     }})
                     self.taskHandles.append(self.bot.loop.call_later(60 * 60 * 12, asyncio.create_task, self.expire_actions(pun['_id'], ctx.guild.id))) # Check in 12 hours, prevents time drifting
-                    diff -= (pun['active_strike_count'] - diff)
+
+                    # Matt would like this to actually recalculate diff to double check, but thats enirely pointless because this logic would simplify to 0:
+                    # new_diff = diff - removed_strikes
```

Comment:
Thanks for the gleeful call out ha. If you wanna just adjust the comment so the math is documented by itself then we are good